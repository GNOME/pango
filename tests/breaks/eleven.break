# Various Emoji sequences
❤️︎︎👨‍🦰👨🏿‍🦱0️⃣🏴󠁵󠁳󠁣󠁡󠁿🇩🇪️
