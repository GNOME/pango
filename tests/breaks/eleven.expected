Text:         ‚Å¶‚ù§‚Å©‚Å¶Ô∏è‚Å© ‚Å¶Ô∏é‚Å© ‚Å¶Ô∏é‚Å©  ‚Å¶üë®‚Å©[0x200d] ‚Å¶ü¶∞‚Å©  ‚Å¶üë®‚Å©‚Å¶üèø‚Å©[0x200d] ‚Å¶ü¶±‚Å©  ‚Å¶0‚Å©‚Å¶Ô∏è‚Å© ‚Å¶‚É£‚Å©  ‚Å¶üè¥‚Å©[0xe0075][0xe0073][0xe0063][0xe0061][0xe007f]  ‚Å¶üá©‚Å©‚Å¶üá™‚Å©‚Å¶Ô∏è‚Å© [0x0a]  
Breaks:     c       lc           lc            lc    lc                                              lc   c      Lc
Whitespace:                                                                                               w      w 
Sentences:  bs                                                                                            e      b 
Words:      b       b            b             bs    be                                              b    b      b 
Graphemes:  b       b            b             b     b                                               b    b      b 
Hyphens:        i i            i             i     i                                                               
